server/server.js

```diff
@@ -74,7 +75,7 @@ class DataSyncServer {
     this.serverConfig = serverConfig
 
     // generate the GraphQL Schema
-    const { schema, dataSources } = await buildSchema(this.models, this.pubsub, this.serverConfig.schemaDirectives)
+    const { schema, dataSources } = await buildSchema(this.models, this.pubsub, this.serverConfig.schemaDirectives, makeExecutableSchema)
```

Comment:
Actually I found out if we do not pass directives to graphql schema build it will fail validation. 
Currently we have @hasRole directive. The way I will suggest to aproach this is that core will have definitions for directives and server itself will provide implementations